packages/fuel-indexer-database/postgres/src/lib.rs

```diff
@@ -633,13 +687,15 @@ pub async fn index_id_for(
     #[cfg(feature = "metrics")]
     METRICS.db.postgres.index_id_for_calls.inc();
 
-    let query = format!(
-        "SELECT id FROM index_registry WHERE namespace = '{namespace}' AND identifier = '{identifier}'",
-    );
-
-    let row = sqlx::query(&query).fetch_one(conn).await?;
+    let row = sqlx::query(
+        r#"select id from index_registry where namespace = $1 and identifier = $2"#,
+    )
+    .bind(namespace)
+    .bind(identifier)
+    .fetch_one(conn)
+    .await?;
 
-    let id: i64 = row.get(0);
+    let id: i64 = row.get("id");
```

Comment:
This is the only place we're referencing by field name instead of field position. Any reason this is done differently?

Reply:
I'm not sure exactly what you mean. But now I'm passing in the name arg and declare a variable with explicit types. Maybe you were (rightly) pointing my lack of consistency in this old commit!

Reply:
using positional args always in latest commit.